gear/gear/global_security_headers.py

```diff
@@ -0,0 +1,11 @@
+from aiohttp import web
+
+
+@web.middleware
+async def global_security_headers_middleware(request: web.Request, handler):
+    response = await handler(request)
+
+    response.headers['Strict-Transport-Security'] = 'max-age=63072000; includeSubDomains;'
+    response.headers['X-Content-Type-Options'] = 'nosniff'
```

Comment:
Is this the full list of security headers? These two are important, but the CSP header does need to be served on all html responses.

Reply:
See also the `web_security_headers` (separate from `api_security_headers`) in web_common.py. For html responses we also add a CSP